doc/user-manual/language/reflection.lagda.rst

```diff
@@ -711,3 +711,34 @@ Example usage:
       defId id-name
 
     unquoteDecl id′ = mkId id′
+
+System Calls
+~~~~~~~~~~~~
+
+It is possible to run system calls as part of a metaprogram, using the ``execTC`` builtin. You can use this feature to implement type providers, or to call external solvers. For instance, the following example calls ``/bin/echo`` from Agda:
+
+.. code-block:: agda
+
+  postulate
+    execTC : (exe : String) (args : List String) (stdIn : String)
```

Comment:
Consider using type synonyms for `String`.

Reply:
I think adding type synonym definitions would obfucate the documentation, rather than enhancing it. The inputs are named, the function is used a few lines down with clearly named outputs, and both inputs and outputs are described in the accompanying text.

Reply:
Ok.